com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ConfigurationEntriesCloudModelBuilder.java

```diff
@@ -25,11 +25,13 @@
     private final String orgName;
     private final String spaceName;
     private final String spaceId;
+    private final String namespace;
 
-    public ConfigurationEntriesCloudModelBuilder(String orgName, String spaceName, String spaceId) {
+    public ConfigurationEntriesCloudModelBuilder(String orgName, String spaceName, String spaceId, String namespace) {
```

Comment:
What do you think to use builder (Immutables)?

Reply:
see comment for ApplicationEnvironmentCloudModelBuilder, it's a good idea